crawler/file_processing.py

```diff
@@ -638,6 +646,38 @@ def update_doc_from_source_plate(
 
         return updated_docs
 
+    def docs_to_insert_updated_with_cog_uk_ids(self, docs_to_insert: List[ModifiedRow]) -> List[ModifiedRow]:
+        """Updates sample records with COG UK IDs.
+
+        Arguments:
+            docs_to_insert {List[ModifiedRow]} -- the sample records to update.
+
+        Returns:
+            List[ModifiedRow] -- the updated samples.
+        """
+        logger.debug("Attempting to update docs with COG UK IDs")
+
+        updated_docs: List[ModifiedRow] = []
+
+        try:
+            prefix = self.centre_config[CENTRE_KEY_PREFIX]
+            count = len(docs_to_insert)
+            cog_uk_ids = generate_baracoda_barcodes(self.config, prefix, count)
+
+            for i, row in enumerate(docs_to_insert):
+                row[FIELD_MONGO_COG_UK_ID] = cog_uk_ids[i]
+                updated_docs.append(row)
+        except Exception as ex:
+            self.logging_collection.add_error(
+                "TYPE 35",
+                f"Failed assigning COG UK IDs to samples in file {self.file_name}",
+            )
```

Comment:
Is this `self.file_name` still relevant after your change from sftp to 🐰 ?

Reply:
No, but this is not the process used by RabbitMQ.  This is an update to file handling.